### ISSUE TITLE
Statement information parsed as transaction description

### ISSUE THREAD

**Issue by ndrsn** [webschik/mt940-js#12]
First of all, thank you for all the work on this! It's already saved me a bunch of time.

That said, I believe there may be a bug in the way statements are parsed, with regards to field type `86`. 

[This definition](https://www2.swift.com/knowledgecentre/publications/usgf_20190719/2.0?topic=con_sfld_Bgxu6448EeiRFupLWiBCUQ_1379610854fld.htm) describes how field `86` is used for extra information pertaining a transaction, when it follows field 61.

However, [this definition](https://www2.swift.com/knowledgecentre/publications/usgf_20190719/2.0?topic=con_sfld_BgxvM448EeiRFupLWiBCUQ_-922688656fld.htm), following field `65` describes it as a field containing extra information on the _statement as a whole_.

I've encountered an issue with `mt940-js` when attempting to read an MT940 similar to the one below (simplified):
```
{1:000001INGBNL2AXXXX00001}
{2:I940INGBNL2AXXXN}
{4:
:20:INGEB
:25:0111111111
:28C:100
:60F:C100722EUR44,89
:61:2001010101C9035,09N078NONREF//20002424669822
/TRCD/00112/
:86:0111111111 V. DE JONG KERKSTRAAT 1154 1234 BW
ENSCHEDE BET.KENM. 1004510036716378 3305330802
AFLOSSINGSTERMIJN 188616 / 1E TERMIJN
:62F:C200101EUR550000,00
:64:C200101EUR550000,00
:65:C200103EUR550000,00
:65:C200101EUR550000,00
:86:/SUM/10/3/308435,72/14435,72/
-}
```

I would expect the transaction description to match

```
"0111111111 V. DE JONG KERKSTRAAT 1154 1234 BW ENSCHEDE BET.KENM. 1004510036716378 3305330802 AFLOSSINGSTERMIJN 188616 / 1E TERMIJN"
```

whereas now it sets it to

```
"/SUM/10/3/308435,72/14435,72/-}"
```

...which is the information related to the statement, not the transaction.

I believe this is because right now no accounting is made for the relative position of field `86` (see [this line](https://github.com/webschik/mt940-js/blob/a09dd06bd2e8c0dba59d2019eacdcccdbf4e764d/src/tags/information-for-account-owner.ts#L43), which indiscriminately sets/overwrites the transaction description.

Can you confirm this is a bug in the implementation?

**Comment by webschik** [comment]
Hi @ndrsn !
Thanks for kind words and your report.
Indeed, it looks like a bug, I will try to check it when I have time. Feel free to open PR ;)

**Comment by ndrsn** [comment]
How about this?

**Comment by ndrsn** [comment]
I've encountered another issue related to this field: when no transactions at all are present in the file, and the parser encounters field `:86:` (like the `/SUM/...` field above), it will throw an error:

```
TypeError: Cannot read property 'description' of undefined

  at Object.close (src/tags/information-for-account-owner.ts:36:139)
  at closeCurrentTag (src/parser.ts:29:19)
  at Object.read (src/parser.ts:73:5)
  at Object.read (src/index.ts:17:19)
```

This is because `state.transactionIndex` will still be set to `-1`. We need to take the possibility of no transactions into account.

I've fixed this in the [latest commit](https://github.com/ndrsn/mt940-js/commit/a11bb08bb9a5c3b62d63379ac1ed0f7033bfaa57) on `master`.
